test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/net/URITest.scala

```diff
@@ -273,6 +279,16 @@ class URITest {
     resTest("http:g", "http:g")
   }
 
+  @Test def should_provide_resolve_issue_2048(): Unit = {
```

Comment:
Consider adding something more descriptive and just linking the bug in a comment. It's nice to understand what is tested without going to the bugtracker.